set.mm

```diff
@@ -649883,6 +649889,35 @@ fixed reference functional determined by this vector (corresponding to
 #*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#
 $)
 
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+  Definitions
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+$)
+
+
+  $c AlgClosure $. $( Algebraic closure $)
+
+  $( Integral elements of a ring. $)
+  calgclosure $a class AlgClosure $.
```

Comment:
Label should be shorter, maybe "calgcl" (this would fit to the label "ax-algclex").

Reply:
Maybe also the class name can be shortened, e.g., "AlgCl"